modules/programs/ssh.nix

```diff
@@ -533,7 +533,10 @@ in {
         '') ++ (map (block: matchBlockStr block.name block.data) matchBlocks))}
 
       Host *
-        ForwardAgent ${lib.hm.booleans.yesNo cfg.forwardAgent}
+        ${
```

Comment:
this is why I like freeform settings: we could have just removed the nix option instead. Is this something you would be interested in impleementing ? 

Now that it exists it makes sense to allow a "null" but with the current implementation it adds an empty line (a recurrent issue throughout nixpkgs). Could you fix that ?